go/interpreter/lfvm/instructions.go

```diff
@@ -250,6 +250,7 @@ func opSstore(c *context) error {
 	cost := tosca.Gas(0)
 	if c.isAtLeast(tosca.R09_Berlin) &&
 		c.context.AccessStorage(c.params.Recipient, key) == tosca.ColdAccess {
+		// we do not call useGas here because it is already checked that there is at least 2300 gas available
```

Comment:
This does not really make sense, because `useGas` is more than checking whether there is enough gas. It also consumes gas. The actual reason why `useGas` is not called here is that several cost factors are summed up and `useGas` is called for the aggregated value once, in line 260.
```suggestion
```

Reply:
while refactoring we decided to not aggregate costs and simply call `c.useGas`. But after that, looking at coverage, we realised of what is explained in this comment. So this is mostly a note to future devs to not try to use `c.useGas` here.